README.md

```diff
@@ -111,99 +194,9 @@ The `plugin:markdown/recommended` config disables these rules in Markdown files:
-            },
-            rules: {
-                "no-console": "off"
-            }
-        }
-    ]
-};
-```
-
-[RFC3](https://github.com/eslint/rfcs/blob/master/designs/2018-processors-improvements/README.md) designed a new processor API to remove these limitations, and the new API was [implemented](https://github.com/eslint/eslint/pull/11552) as part of ESLint v6.
-`eslint-plugin-markdown` v2 uses this new API.
-
-```bash
-$ npm install --save-dev eslint@latest eslint-plugin-markdown@latest
-```
-
-All of the Markdown file extensions that were previously hard-coded are now fully configurable in `.eslintrc.js`.
-Use the new `processor` option to apply the `markdown/markdown` processor on any Markdown documents matching a `files` pattern.
-Each fenced code block inside a Markdown document has a virtual filename appended to the Markdown file's path.
-The virtual filename's extension will match the fenced code block's syntax tag, so for example, <code>```js</code> code blocks in <code>README.md</code> would match <code>README.md/*.js</code>.
-
-```js
-// eslintrc.js for eslint-plugin-markdown v2
-module.exports = {
-    plugins: ["markdown"],
-    overrides: [
-        {
-            // In v2, explicitly apply eslint-plugin-markdown's `markdown`
-            // processor on any Markdown files you want to lint.
-            files: ["**/*.md"],
-            processor: "markdown/markdown"
-        },
-        {
-            // In v2, configuration for fenced code blocks is separate from the
-            // containing Markdown file. Each code block has a virtual filename
-            // appended to the Markdown file's path.
-            files: ["**/*.md/*.js"],
-            // Configuration for fenced code blocks goes with the override for
-            // the code block's virtual filename, for example:
-            parserOptions: {
-                ecmaFeatures: {
-                    impliedStrict: true
-                }
-            },
-            rules: {
-                "no-console": "off"
-            }
-        }
-    ]
-};
-```
-
-If you need to precisely mimic the behavior of v1 with the hard-coded Markdown extensions and fenced code block syntaxes, you can use those as glob patterns in `overrides[].files`:
-
-```js
-// eslintrc.js for v2 mimicking v1 behavior
-module.exports = {
-    plugins: ["markdown"],
-    overrides: [
-        {
-            files: ["**/*.{md,mkdn,mdown,markdown}"],
-            processor: "markdown/markdown"
-        },
-        {
-            files: ["**/*.{md,mkdn,mdown,markdown}/*.{js,javascript,jsx,node}"]
-            // ...
-        }
-    ]
-};
-```
-
 ### Running
 
-#### ESLint v7
+#### ESLint v7+
 
 You can run ESLint as usual and do not need to use the `--ext` option.
 ESLint v7 [automatically lints file extensions specified in `overrides[].files` patterns in config files](https://github.com/eslint/rfcs/blob/0253e3a95511c65d622eaa387eb73f824249b467/designs/2019-additional-lint-targets/README.md).
```

Comment:
This part of the documentation also needs to be updated(or can be removed 🤔)：
* `--ext` and `overrides` are no longer supported in flat configs